confluent_kafka/error.py

```diff
@@ -30,23 +29,11 @@ class ConsumeError(Exception):
     Args:
         error (KafkaError): The error that occurred.
```

Comment:
Error seems to be an error code now, not KafkaError.
Both doc string and param name should probably be changed.